server/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -128,13 +128,13 @@ object CORS {
           }
         case _ =>
           // This request is out of scope for CORS
-          http(req)
+          http
       }
     }
 
-  def httpRoutes[F[_]: Monad](httpRoutes: HttpRoutes[F]): HttpRoutes[F] =
+  def httpRoutes[F[_]](httpRoutes: HttpRoutes[F])(implicit F: Monad[F]): HttpRoutes[F] =
     apply(httpRoutes)
 
-  def httpApp[F[_]: Applicative](httpApp: HttpApp[F]): HttpApp[F] =
+  def httpApp[F[_]](httpApp: HttpApp[F])(implicit F: Monad[F]): HttpApp[F] =
```

Comment:
Why did you strengthen this constraint?

Reply:
Because `apply` needs a `Monad` now, and `Kleisli` doesn't have one unless `F` has one.